OpenRA.Mods.HV/Traits/SupportPowers/TeleportPower.cs

```diff
@@ -56,6 +57,23 @@ sealed class TeleportPowerInfo : SupportPowerInfo
 		[Desc("Cursor to display when the targeted area is blocked.")]
 		public readonly string TargetBlockedCursor = "move-blocked";
 
+		[Desc("Maximum range from televator to be able to teleport.")]
+		public readonly int RangeFromTelevatorsSquared = 100;
```

Comment:
```suggestion
		public readonly WDist RangeFromTelevators = 100;
```
to not leak an implementation detail into game rules.